practice/bio/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+
+<html lang= "en">
+
+  <head>
+    <meta charset="utf-8">
+    <title>Bio</title>
+    <link rel="stylesheet" href="site.css" />
+  </head>
+
+  <body>
+    <header>
+      <h1>John Cena Bio</h1>
+    </header>
+    <nav>
+    </nav>
+    <section>
+      <p>
+        John Felix Anthony Cena Jr. (born April 23, 1977) is an American professional wrestler, rapper, actor, and reality television show host signed to WWE on the SmackDown brand.
+      </p>
+      <img src="johncena.jpg">
+    </section>
+    <section>
+      <h2>Places lived in</h2>
+      <p>
+        <ul>
+          <li>West Newburry, Massachusetts</li>
+          <li>Tampa, Florida</li>
+        </ul>
+      </p>
+    </section>
+    <section>
+      <blockquote class="quote">
+        <em>I don't doll myself up for TV because I want people to accept me for who I am.</em>
```

Comment:
Same feedback as before. Don't use a tag just for it's style, edit the CSS directly to get italics on quotes.